RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 * Better support for creating or editing posts while offline. Posts can be saved while offline and they will be automatically uploaded (or published) when the device is back online.
 * Me view: fix issue where view was blank when logging in with a self-hosted site.
 * Block Editor: Added support for image alignment options.
+* Stats: each Period and Post stat now loads independently.
```

Comment:
Do we want to add this to release notes now since it's not going to be public in this release?